src/components/VContentSwitcher/VSearchTypes.vue

```diff
@@ -18,11 +26,41 @@
       :selected="item === activeItem"
       @click="handleClick(item)"
     />
+    <div
+      v-if="content.additionalTypes && content.additionalTypes.length > 0"
+      class="mt-2"
+      :class="{
+        'bg-dark-charcoal-06 border-t border-dark-charcoal-20': !bordered,
+      }"
+    >
+      <h4
+        :class="{
+          'ps-0': !bordered,
+          'ps-6': !bordered,
+        }"
+        class="text-sr pt-6 pe-6 pb-4 uppercase font-semibold"
+      >
+        {{ $t('search-type.additional') }}
+      </h4>
+      <VSearchTypeItem
+        v-for="(item, idx) in content.additionalTypes"
+        :key="item"
+        class="md:mb-1"
+        :item="item"
+        :item-id="idx"
+        :icon="content.icons[item]"
+        :use-links="useLinks"
+        :selected="item === activeItem"
+        @click="handleClick(item)"
+      />
+    </div>
```

Comment:
This seems duplicated, there should be a way for us to reuse the upper-bit of code.

Reply:
That would be nice, but I also don't think this is worth abstracting into a standalone compoent or something like that. What do you suggest?

Reply:
Yeah, I wasn't suggesting making a separate component for it, that would be overkill. My idea was to use groups to combine regular and additional media types. I've implemented my version in PR #1204.